packs/feats/ancestry/shared-ancestry-feats/skillful-tail.json

```
@@ -0,0 +1,75 @@
+{
+    "_id": "aVRuchEAJIvnd70k",
+    "folder": "843OlqYqFtC36q56",
+    "img": "icons/sundries/books/book-red-exclamation.webp",
+    "name": "Skillful Tail",
+    "system": {
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "category": "ancestry",
+        "description": {
+            "value": "<p><strong>Geniekin -</strong> You were born with a tail, and with practice you've learned how to perform simple Interact actions with it, such as opening an unlocked door. Your tail can't perform actions that require fingers or significant manual dexterity, including any action that would require a check to accomplish, and you can't use it to hold items.</p><hr /><p><strong>Nephilim -</strong> You've always had a tail, but with practice, you've learned to use it for more than signaling your mood. You can perform simple Interact actions with your tail, such as opening an unlocked door. Your tail can't perform actions that require fingers or significant manual dexterity, including any action that would require a check to accomplish, and you can't use it to hold items.</p>"
```

```suggestion
            "value": "<p><strong>Geniekin</strong> You were born with a tail, and with practice you've learned how to perform simple Interact actions with it, such as opening an unlocked door. Your tail can't perform actions that require fingers or significant manual dexterity, including any action that would require a check to accomplish, and you can't use it to hold items.</p><hr /><p><strong>Nephilim</strong> You've always had a tail, but with practice, you've learned to use it for more than signaling your mood. You can perform simple Interact actions with your tail, such as opening an unlocked door. Your tail can't perform actions that require fingers or significant manual dexterity, including any action that would require a check to accomplish, and you can't use it to hold items.</p>"
```
I think we can skip the dash, like how shared class features are done